appinventor/appengine/src/com/google/appinventor/shared/rpc/semweb/SemWebConstants.properties

```diff
@@ -1,3 +1,7 @@
 defaultEndpoint = http://dbpedia.org/sparql
 #ontologies = http://orion.tw.rpi.edu/~pattoe/dbpedia_3.8.owl,http://xmlns.com/foaf/0.1/
-ontologies = http://xmlns.com/foaf/0.1/,http://hxl.humanitarianresponse.info/ns/hxl.ttl,http://orion.tw.rpi.edu/~pattoe/moac.ttl,https://download.bio2rdf.org/files/current/drugbank/drugbank.schema.owl,http://purl.org/dc/terms/,https://www.w3.org/2001/sw/RDFCore/Schema/200212bwm/rdfs-namespace.xml,https://schema.org/version/latest/schemaorg-current-http.ttl,https://projects.cs.dal.ca/niche/sleepapnea.owl
+
+#not found: http://orion.tw.rpi.edu/~pattoe/moac.ttl
+#syntax errors: https://www.w3.org/2001/sw/RDFCore/Schema/200212bwm/rdfs-namespace.xml
+
+ontologies = http://xmlns.com/foaf/0.1/,http://hxl.humanitarianresponse.info/ns/hxl.ttl,https://download.bio2rdf.org/files/current/drugbank/drugbank.schema.owl,http://purl.org/dc/terms/,https://schema.org/version/latest/schemaorg-current-http.ttl,https://projects.cs.dal.ca/niche/sleepapnea.owl
```

Comment:
I'd recommend against removing the rdfs-namespace entry as it does work (although it's not valid XML and so an error is thrown after the fact) and removing it basically removes all of the rdfs terms from autocomplete.

Reply:
> I'd recommend against removing the rdfs-namespace entry as it does work (although it's not valid XML and so an error is thrown after the fact)

That's why I added it under syntax errors :-) IMO it's not good practice to load malformed data by default 

Also note that some of the terms are no longer correct - Statement and related terms belong under the rdf namespace, not the rdfs one

Why not just add the official namespace and its schema file at www.w3.org/2000/01/rdf-schema# - I amended my PR